utils/avd_install.sh

```diff
@@ -2,29 +2,30 @@
 
 set -e
 
-SDKDIR=$PWD/.android-sdk
-export ANDROID_SDK_ROOT=${SDKDIR}
-export ANDROID_HOME=${SDKDIR}
-export ANDROID_AVD_HOME=${SDKDIR}/avd
+if [[ -n "${ANDROID_HOME}" ]]; then
+    echo "ANDROID_HOME is already set. This script is only for a local installation of the Android SDK."
+    exit 1
+fi
 
-mkdir ${SDKDIR}
-mkdir ${SDKDIR}/cmdline-tools
+export ANDROID_HOME="$PWD/.android-sdk"
+
+mkdir ${ANDROID_HOME}
+mkdir ${ANDROID_HOME}/cmdline-tools
 
 echo Downloading Android SDK...
-cd ${SDKDIR}/cmdline-tools
-COMMAND_LINE_TOOLS_ZIP=${SDKDIR}/commandlinetools.zip
-# https://developer.android.com/studio#command-tools
-curl https://dl.google.com/android/repository/commandlinetools-mac-8512546_latest.zip -o ${COMMAND_LINE_TOOLS_ZIP}
-unzip ${COMMAND_LINE_TOOLS_ZIP}
+cd ${ANDROID_HOME}/cmdline-tools
+COMMAND_LINE_TOOLS_ZIP=${ANDROID_HOME}/commandlinetools.zip
+# https://developer.android.com/studio
+curl https://dl.google.com/android/repository/commandlinetools-mac-11076708_latest.zip -o ${COMMAND_LINE_TOOLS_ZIP}
+unzip ${COMMAND_LINE_TOOLS_ZIP} 
 rm ${COMMAND_LINE_TOOLS_ZIP}
 mv cmdline-tools latest
-ln -s ${SDKDIR}/cmdline-tools/latest ${SDKDIR}/tools
 
 echo Installing emulator...
-yes | ${ANDROID_HOME}/tools/bin/sdkmanager --install platform-tools emulator
+yes | ${ANDROID_HOME}/cmdline-tools/latest/bin/sdkmanager --install platform-tools emulator
```

Comment:
In theory we could remove line 25-31, since we do the work / ensure it now in adb_recreate.sh.